Packs/RsaNetWitnessPacketsAndLogs/Integrations/RSANetWitnessPacketsAndLogs/RSANetWitnessPacketsAndLogs.js

```diff
@@ -5,13 +5,17 @@ var RESPONSE_TYPES = {
     'xml': 'text/xml',
     'octet-stream': 'application/octet-stream'
 };
-var SSL_PORT = '56105';
-var NON_SSL_PORT = '50105';
+var API_REST_PORT = '50105';
 var LAST_HOURS = 'lastHours';
 var LAST_MINUTES = 'lastMinutes';
 var QUERY = 'query';
 var USER_NAME = params.user_creds ? params.user_creds.identifier : params.username;
-var PASSWORD = (params.user_creds || params).password;
+
+var PASSWORD =  params.user_cred ? params.user_creds.password : params.password;
+if ( params.username.credentials.user != "" ){
+    USER_NAME = params.username.credentials.user;
+    PASSWORD = params.username.credentials.password;
+}
```

Comment:
Please revert to the way it was - Doesn't work that way for you?
```suggestion
var PASSWORD = (params.user_creds || params).password;

```